libr/core/disasm.c

```diff
@@ -928,7 +928,7 @@ static void handle_show_functions(RCore *core, RDisasmState *ds) {
 	ds->stackptr = 0;
 	if (ds->vars) {
 		char spaces[32];
-		RList *args = r_anal_var_list (core->anal, f, 'a');
+		RList *args = r_anal_var_list (core->anal, f,f->call==R_ANAL_CC_TYPE_FASTCALL?'A':'a');
```

Comment:
always put a space after every : or ? and around ==. You can add parenthesis or move that conditional right after the `char spaces[32]` if you want less width